packages/core/src/components/Menu/Menu/utils/utils.ts

```diff
@@ -2,7 +2,7 @@ import { ReactElement } from "react";
 import { MenuChild } from "../MenuConstants";
 
 export function isMenuChildSelectable(child: MenuChild): boolean {
-  return !!child.type.isSelectable && !child.props.disabled;
+  return !!child.type.isSelectable;
```

Comment:
Since we only want it to be selectable if it is disabled and has a tooltip (but should not be selectable when it is disabled without a tooltip), the check for disabled should stay, and there should be an additional check if has tooltip.